sdk/storage/azure-storage-file-share/azure/storage/fileshare/_share_client.py

```diff
@@ -314,6 +314,11 @@ def create_share(self, **kwargs):
 
         :keyword int timeout:
             The timeout parameter is expressed in seconds.
+        :keyword enabled_protocols: Protocols to enable on the share.
```

Comment:
enabled_protocols should be a list, else it's enabled_protocol.
If I remember properly, Sean said currently only a single protocol, while later we will support bunch

Reply:
https://github.com/Azure/azure-sdk-for-java/pull/17239/files#diff-fb6910d874ec538dd2a452d3aaca8b72cc17c8f9fb70b23f4a7fd020bb6c632eR57-R68